### PR TITLE
Fix inconsistent spacing in error messages and translations

### LINKED ISSUE #2376: Inconsistent spacing in error messages and translations
1. [x] src/soundbase.cpp needs a fix in code and English `<source>` texts in all *.ts files: https://github.com/jamulussoftware/jamulus/blob/366dd0bd1f6093d2f87081e50bd1bdd0aa4a70d7/src/soundbase.cpp#L159
1. [x] Below `grep` finds some more consistencies (including fr_FR and it_IT related to the previous item)

```
$ grep -hv vanished src/res/translation/*.ts | grep -P '<(source|translation).*[.](?!([& <.:,]|io|org|net|sourceforge|v\.|e\.|p\.v\.|u\.b\.|b\. |B\. |h\. |ex\.|g\.|0 |B\.|$))'  | sort -u
        <source>Click the button to open the dialog that allows the main recording directory to be selected.The chosen value must exist and be writeable (allow creation of sub-directories by the user Jamulus is running as). </source>
        <source>Controls the relative levels of the left and right local audio channels. For a mono signal it acts as a pan between the two channels.For example, if a microphone is connected to the right input channel and an instrument is connected to the left input channel which is much louder than the microphone, move the audio fader in a direction where the label above the fader shows %1, where %2 is the current attenuation indicator.</source>
        <source>Controls the relative levels of the left and right local audio channels. For a mono signal it acts as a pan between the two channels.For example, if a microphone is connected to the right input channel and an instrument is connected to the left input channel which is much louder than the microphone, move the audio fader in a direction where the label above the fader shows </source>
        <source>For proper usage of the application, you should not hear your singing/instrument through the loudspeaker or your headphone when the software is not connected.This can be achieved by muting your input audio channel in the Playback mixer (not the Recording mixer!).</source>
        <source>If you know the IP address or URL of a server, you can connect to it using the Server name/Address field. An optional port number can be added after the IP address or URL using a colon as a separator, e.g, %1. The field will also show a list of the most recently used server addresses.</source>
        <source>If you know the IP address or URL of a server, you can connect to it using the Server name/Address field. An optional port number can be added after the IP address or URL using a colon as a separator, e.g, example.org:</source>
        <source> server software must be set here. An optional port number can be added after the IP address or URL using a colon as a separator, e.g, example.org:</source>
        <source> server software must be set here. An optional port number can be added after the IP address or URL using a comma as a separator, e.g, example.org:</source>
        <source>The previously selected audio device is no longer available or the driver has changed to an incompatible state.We&apos;ll attempt to find a valid audio device, but this new audio device may cause feedback.Before connecting to a server, please check your audio device settings.</source>
        <source>This setting allows you to increase your input signal level by factors up to 10 (+20dB).If your sound is too quiet, first try to increase the level by getting closer to the microphone, adjusting your sound equipment or increasing levels in your operating system&apos;s input settings. Only if this fails, set a factor here. If your sound is too loud, sounds distorted and is clipping, this option will not help. Do not use it. The distortion will still be there. Instead, decrease your input level by getting farther away from your microphone, adjusting your sound equipment or by decreasing your operating system&apos;s input settings.</source>
        <translation>Ce paramètre vous permet d&apos;augmenter le niveau de votre signal d&apos;entrée par des facteurs allant jusqu&apos;à 10 (+20dB).Si votre son est trop faible, essayez d&apos;abord d&apos;augmenter le niveau en vous rapprochant du microphone, en réglant votre équipement de sonorisation ou en augmentant les niveaux dans les paramètres d&apos;entrée de votre système d&apos;exploitation. Ce n&apos;est qu&apos;en cas d&apos;échec que vous pouvez définir un facteur ici. Si votre son est trop fort, s&apos;il est déformé et s&apos;il y a de l&apos;écrêtage, cette option ne vous aidera pas. Ne l&apos;utilisez pas. La distorsion sera toujours présente. Diminuez plutôt votre niveau d&apos;entrée en vous éloignant de votre microphone, en réglant votre équipement de sonorisation ou en diminuant les paramètres d&apos;entrée de votre système d&apos;exploitation.</translation>
        <translation>Il dispositivo audio selezionato in precedenza non è più disponibile o il driver è passato a uno stato incompatibile.Cercheremo di trovare un dispositivo audio valido.Prima di connetterti a un server, controlla le connessioni sul tuo dispositivo audio dalle impostazioni.</translation>
        <translation>Le périphérique audio précédemment sélectionné n&apos;est plus disponible ou le pilote est passé à un état incompatible.Nous allons essayer de trouver un périphérique audio valide, mais ce nouveau périphérique audio peut provoquer un effet larsen.Avant de vous connecter à un serveur, veuillez vérifier les paramètres de votre périphérique audio.</translation>
        <translation>O gravador falhou ao iniciar.Verifique o espaço em disco disponível e as permissões e tente novamente. Erro: </translation>
        <translation>Questa impostazione consente di aumentare il livello del segnale in ingresso tramite fattori di moltiplicazione di guadagno fino a 10 (ovvero + 20dB) .Se il suono è troppo basso provare, prima ad aumentare il guadagno, di avvicinarsi al microfono, regolare i livelli della scheda audio o regolare i livelli nelle impostazioni di input del sistema. Se non si riesce ad ottenere un livello di ingresso soddisfacente, impostare un fattore di guadagno adeguato. Se il suono è gia troppo forte risulterà distorto ed in clipping, questa opzione in questo caso non ti sarà utile. In questi casi dovrai diminuire il livello di ingresso: allontanati dal microfono, regola la scheda audio o regola i livelli delle impostazioni di ingresso del sistema operativo.</translation>
```

Note: We should be able to do this via manual edits. No need for PRs by translators.

### PR DESCRIPTION
**Pull request by hoffie**
**Short description of changes**

- Multiple English source text lacked spaces after dots. This PR fixes the source code and the entry in all .ts files.
- Some translations lacked spaces after dots independent of the previous item.

As this PR might get conflicts, I'm documenting the sed calls here for easier reproduction (note: source code and translated strings were edited manually though!):

```
sed -re 's/The previously selected audio device is no longer available or the driver has changed to an incompatible state\.We&apos;ll attempt to find a valid audio device, but this new audio device may cause feedback\.Before connecting to a server, please check your audio device settings\./The previously selected audio device is no longer available or the driver has changed to an incompatible state. We\&apos;ll attempt to find a valid audio device, but this new audio device may cause feedback. Before connecting to a server, please check your audio device settings./' -i src/res/translation/*.ts

sed -re 's|For proper usage of the application, you should not hear your singing/instrument through the loudspeaker or your headphone when the software is not connected\.This can be achieved by muting your input audio channel in the Playback mixer \(not the Recording mixer!\)\.|For proper usage of the application, you should not hear your singing/instrument through the loudspeaker or your headphone when the software is not connected. This can be achieved by muting your input audio channel in the Playback mixer (not the Recording mixer!).|' -i src/res/translation/*.ts

sed -re 's|\.(If your sound is too quiet)|. \1|' -i src/res/translation/*.ts

sed -re 's|(the two channels\.)(For example)|\1 \2|' -i src/res/translation/*.ts

sed -re 's|(using a colon as a separator, e\.g),|\1.|' -i src/res/translation/*.ts

sed -re 's|(directory to be selected\.)(The)|\1 \2|' -i src/res/translation/*.ts
```

`lupdate` has been used to confirm that the manual edits did not confuse the Qt tools.

**Context: Fixes an issue?**

Fixes #2376 

**Does this change need documentation? What needs to be documented and how?**

No.
CHANGELOG: SKIP

**Status of this Pull Request**

Ready for review, but not for merge (see below).

**What is missing until this pull request can be merged?**

Should wait until each and every 3.8.2 translation PR has been merged. Check the milestone.

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I tested my code and it does what I want
- [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I've filled all the content above

### CODE REVIEW

File: src/res/translation/translation_fr_FR.ts [hunk 1]
```diff
@@ -1517,7 +1517,7 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="344"/>
         <source>This setting allows you to increase your input signal level by factors up to 10 (+20dB).If your sound is too quiet, first try to increase the level by getting closer to the microphone, adjusting your sound equipment or increasing levels in your operating system&apos;s input settings. Only if this fails, set a factor here. If your sound is too loud, sounds distorted and is clipping, this option will not help. Do not use it. The distortion will still be there. Instead, decrease your input level by getting farther away from your microphone, adjusting your sound equipment or by decreasing your operating system&apos;s input settings.</source>
```
Comment (softins) [inline src/res/translation/translation_fr_FR.ts:1519]: This line too, and its source and other translations.

**Comment by softins** [comment]
Once the source files and ts files have been fixed manually and committed, a good check would be to rerun `lupdate` and check the ts files don't get changed.

File: src/res/translation/translation_fr_FR.ts (hunk 1 above)
Comment (hoffie) [reply src/res/translation/translation_fr_FR.ts]: Thanks. Done, along with other source fixes.

**Comment by hoffie** [comment]
> Once the source files and ts files have been fixed manually and committed, a good check would be to rerun `lupdate` and check the ts files don't get changed.

Good idea -- done, it doesn't show any changes for me.

**Review by softins — APPROVED** [review]
Looks good. Thanks for doing this!

**Review by pljones — APPROVED** [review]

**Pull request merged**